app/views/view_components/preview.html.erb

```diff
@@ -1,13 +1,11 @@
-<% if @render_args[:component] %>
-  <% if ViewComponent::Base.render_monkey_patch_enabled || Rails.version.to_f >= 6.1 %>
-    <%= render(@render_args[:component], @render_args[:args], &@render_args[:block]) %>
-  <% else %>
-    <%= render_component(@render_args[:component], &@render_args[:block]) %>
-  <% end %>
+<% @render_args ||= render_args %>
```

Comment:
What necessitated this change?

Reply:
This was added because `render_args` the method needed to be called to fetch the render_args. Without this, the system test helpers would not be able to determine what the `@render_args` should be when generating the temporary template to test against.